### ISSUE TITLE
SVGO config not being used

### ISSUE THREAD

**Issue by jan-dh** [freshheads/svg-spritemap-webpack-plugin#12]
The code below outputs an SVG that is minified, with double title attributes and no prefix. Tried on version 1.0.3

```
new SVGSpritemapPlugin({
	src:path.resolve(__dirname,basePath + '/img/icons-svg/**/*.svg'),
	filename: 'img/sprite.svg',
	svgo:{
		removeTitle: false,
		minify:false,
		removeXMLNS:true,
		prefix: 'svg-icon-',
	},
	svg4everybody:true,
}),

**Comment by cascornelissen** [comment]
The `prefix` option is not related to `svgo`, it should be added as a root key (just like `filename`, etc.) 

And what exactly do you mean with double title attributes? Could you provide an example?

**Comment by jan-dh** [comment]
The plugin outputs the svg's in the sprite as such (minified)

```
<symbol id="sprite-twitter" viewBox="0 0 18 14.62">
		<title>twitter</title>
		<title>twitter</title>
```

**Comment by jacubsmith** [comment]
Try using the following instead:

`new SVGSpritemapPlugin({
	src:path.resolve(__dirname,basePath + '/img/icons-svg/**/*.svg'),
	filename: 'img/sprite.svg',
	svgo:{
		plugins: [{
			removeTitle: false,
			minify:false,
			removeXMLNS:true,
		}]
	},
	svg4everybody:true,
}), `

**Comment by cascornelissen** [comment]
I just tested this with @jacubsmith's settings and I can't reproduce the double title element.

If anyone else runs into a similar issue please provide a complete example and I'll reopen the issue.

**Comment by cascornelissen** [comment]
I've been able to reproduce the double `title` element bug, no idea why I didn't run into this any earlier but it'll be fixed in `1.0.4`!
